### PR TITLE
Mobile menu 

### PR DESCRIPTION
**Pull request by DevKaranJ**
This pull request implements the mobile menu for the portfolio website. The mobile menu appears when the user clicks (or taps) the hamburger button and disappears when the user clicks (or taps) the close (X) button. The mobile menu also disappears when the user clicks (or taps) any of the mobile menu options. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

****

### CODE REVIEW

**Comment by DevKaranJ** [comment]
**Hello Reviewer, I have made changes to the style of my portfolio website. All the previous functionality is still present, and I have ensured that everything exists in the GitHub repository. Kindly review the new style and let me know if any improvements are needed. Thank you!**

File: index.html [hunk 1]
```diff
@@ -17,11 +17,21 @@
...
           </ul>
         </nav>
+        <div id="menuContent" class="menu-content">
+          <button class="close-button" type="button">
+            <img src="./assets/Buttons/mobileMenuClose.png" alt="Close" class="close-icon">
+          </button>
+          <ul class="mobile-menu" id="menu">
+            <li class="menuItem"><a href="#Portfolio" class="menu-target">Portfolio</a></li>
+            <li class="menuItem"><a href="#about" class="menu-target">About</a></li>
+            <li class="menuItem"><a href="#contact" class="menu-target">Contact</a></li>
```
Comment (shahadat3669) [inline index.html]: 👍 You've successfully implemented the mobile menu functionality, and it functions well. However, there are some adjustments needed to align it with the design:

- The font styles look great, but the color doesn't match the design. Please ensure that you update the code to match the design's specified color.

- For each item in the menu, there should be a border-bottom as shown in the design. Make sure to follow the design closely and apply the border as required.

- Additionally, I recommend you adjust the gap for the menu items to match the design as closely as possible. Following the design guidelines for gap and spacing will enhance the visual coherence of your mobile menu.

Group 58 (1)

By addressing these adjustments, your mobile menu will be visually consistent with the design, providing a more polished and cohesive user experience. Keep up the good work, and I look forward to seeing your mobile menu perfectly aligned with the design! 🌟🍔🚀

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @DevKaranJ, @ayaderaghul , 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.



## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ No linter Errors
✅ Descriptive PR Title
✅ Descriptive PR Description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] [Optional] 👍 While your focus is on the mobile version, I'd like to encourage you to also update the desktop version nav items' styles to match the design. This extra effort will enhance the overall consistency and visual appeal of your codebase. Keep up the great work, and I'm confident your attention to detail will lead to an even more polished project! 🌟📱💻

image


Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @DevKaranJ, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**